packages/js-commons/src/local-files.ts

```diff
@@ -68,7 +68,13 @@ export const getLocalApplicationTransporter = (
           } else {
             return;
           }
-          const content = await readFile(filePath, { encoding: "utf-8" });
```

Comment:
`filePath` here refers to the location of the YAML metadata file. The actual asset file location will be either have to be `AssetDTO.publicUrl` or a new attribute that you add.